src/main/resources/assets/api-docs.yaml

```diff
@@ -1742,6 +1742,17 @@ paths:
       summary: Manage Data Access Requests V2
       description: |
         Returns all the elections created for Data Access Requests relevant to the current user.
+      parameters:
+        - name: roleName
+          in: query
+          description: An optional roleName that the user is making the request as. A user can have multiple roles, so adding this parameter will filter the list accordingly.
+          required: false
+          schema:
+            type: String
+            nullable: true
+            enum:
+              - SigningOfficial
+              - null
```

Comment:
We don't need the null value here.

Reply:
If you try and use that, it tries to send `null` as a role name, which obviously fails.